Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -236,500 +139,242 @@ public override void OnInspectorGUI()
-                        var controllerHandedness = mixedRealityControllerMapping.FindPropertyRelative("handedness");
-                        var useDefaultModel = mixedRealityControllerMapping.FindPropertyRelative("useDefaultModel");
-                        var controllerModel = mixedRealityControllerMapping.FindPropertyRelative("overrideModel");
-                        var interactionsList = mixedRealityControllerMapping.FindPropertyRelative("interactions");
-                        var useCustomInteractionMappings = mixedRealityControllerMapping.FindPropertyRelative("useCustomInteractionMappings");
-
-                        EditorGUI.indentLevel++;
-                        EditorGUIUtility.labelWidth = 128f;
-
-                        EditorGUI.BeginChangeCheck();
-                        EditorGUILayout.PropertyField(controllerTypeProperty);
-                        EditorGUILayout.PropertyField(controllerHandedness);
-                        EditorGUIUtility.labelWidth = 224f;
-
-                        if (EditorGUI.EndChangeCheck())
-                        {
-                            serializedObject.ApplyModifiedProperties();
-
-                            if (thisProfile.MixedRealityControllerMappingProfiles[i].ControllerType.Type == null)
-                            {
-                                controllerHandedness.intValue = 0;
-                            }
-
-                            // Only allow custom interaction mappings on generic controller types
-                            var controllerType = thisProfile.MixedRealityControllerMappingProfiles[i].ControllerType.Type;
-                            useCustomInteractionMappings.boolValue = controllerType == typeof(GenericUnityController) ||
-                                                                     controllerType == typeof(GenericOpenVRController);
-                            interactionsList.ClearArray();
-                            serializedObject.ApplyModifiedProperties();
-                            thisProfile.MixedRealityControllerMappingProfiles[i].SetDefaultInteractionMapping();
-                            serializedObject.ApplyModifiedProperties();
-
-                            // Bail so we can execute our controller type set command.
-                            EditorGUI.indentLevel--;
-                            GUILayout.EndVertical();
-                            GUILayout.Space(12f);
-                            GUILayout.EndVertical();
-                            GUILayout.EndVertical();
-                            return;
-                        }
-
-                        EditorGUIUtility.labelWidth = 128f;
-                        EditorGUILayout.PropertyField(useDefaultModel);
-
-                        if (!useDefaultModel.boolValue)
-                        {
-                            EditorGUILayout.PropertyField(controllerModel);
```

Comment:
Is the controller-specific model override field intentionally no longer displayed for all non-generic controllers?